package.json

```diff
@@ -96,6 +96,7 @@
     "isomorphic-fetch": "2.2.1",
     "luxon": "1.2.0",
     "markdown-it": "^8.4.1",
+    "mockdate": "^2.0.2",
```

Comment:
Already pointed on another PR but just as a note to remember, this should go to  `devDeps` :slightly_smiling_face: